kie-wb/kie-wb-webapp/src/main/java/org/kie/workbench/backend/DashbuilderBootstrap.java

```diff
@@ -57,6 +57,11 @@
     @Inject
     protected DataSetDefRegistry dataSetDefRegistry;
 
+
+    public void setDataSetDefRegistry(DataSetDefRegistry dataSetDefRegistry){
```

Comment:
If you use @InjectMocks in the test, Mockito would that care of injecting an instance of DataSetDefRegistry. That way this method wouldn' t be necessary.